code/modules/antagonists/fugitive/fugitive.dm

```diff
@@ -0,0 +1,88 @@
+
+
+/datum/antagonist/fugitive
+	name = "Fugitive"
+	//show_in_antagpanel = FALSE //remove this later- they are event specific. this is 100% for testing
```

Comment:
This still in and pr is marked as testmerge ready.

Reply:
will remove, good reminder